src/utils/collapse-imports.js

```diff
@@ -0,0 +1,39 @@
+// @flow
+import type {
+  BabelPath,
+  Program,
+  ImportDeclaration,
+} from '@ganemone/babel-flow-types';
+
+export function collapseImports(
+  program: BabelPath<Program>
+): BabelPath<Program> {
+  // $FlowFixMe
+  const importStatements: Array<ImportDeclaration> = program.node.body.filter(
+    item => {
+      return item.type === 'ImportDeclaration';
```

Comment:
Should we case on Flow type imports here?  This looks like it would currently collapse both normal and type imports together:

```
import {A} from 'pckg';
import type {AType} from 'pckg';

=>

import {A, AType} from 'pckg';
```

Or

```
import type {AType} from 'pckg';
import {A} from 'pckg';

=>

import type {AType, A} from 'pckg';
```